src/browser/api_protocol/transport_strategy/elipc_strategy.ts

```diff
@@ -191,7 +191,10 @@ export class ElipcStrategy extends ApiTransportBase<MessagePackage> {
             /* tslint:disable: max-line-length */
             //message payload might contain sensitive data, mask it.
             const disableIabSecureLogging = coreState.getAppObjByUuid(opts.uuid)._options.disableIabSecureLogging;
-            const replacer = (!disableIabSecureLogging && (data.action === 'publish-message' || data.action === 'send-message')) ? this.payloadReplacer : null;
+            let replacer = (!disableIabSecureLogging && (data.action === 'publish-message' || data.action === 'send-message')) ? this.payloadReplacer : null;
```

Comment:
Do we need to do this here as well https://github.com/HadoukenIO/core/blob/develop/src/browser/api_protocol/transport_strategy/ws_strategy.ts#L88

Reply:
I don't see any password logged.  Do we have any login for it? Goldman said no.

Reply:
Gotcha. Just noticed it had similar code so wanted to check. 👍